jetty-core/jetty-websocket/jetty-websocket-jetty-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrameHandler.java

```diff
@@ -277,8 +277,27 @@ private void notifyOnClose(CloseStatus closeStatus, Callback callback)
         try
         {
             if (closeHandle != null)
-                closeHandle.invoke(closeStatus.getCode(), closeStatus.getReason());
-            callback.succeeded();
+            {
+                var apiCallback = new org.eclipse.jetty.websocket.api.Callback()
```

Comment:
Is there a reason to not use `from()`? It would make the code more readable.

Reply:
now using `Callback.from()`